example/src/Screens/BottomTabsAnimated.tsx

```diff
@@ -0,0 +1,142 @@
+
+type BottomTabParams = {
+  TabStack: NavigatorScreenParams<SimpleStackParams>;
+  TabAlbums: undefined;
+  TabContacts: undefined;
+  TabChat: undefined;
+};
+
+const AlbumsScreen = () => {
+  const headerHeight = useHeaderHeight();
+  const tabBarHeight = useBottomTabBarHeight();
+  const isFocused = useIsFocused();
+
+  return (
+    <>
+      {isFocused && <StatusBar barStyle="light-content" />}
+      <ScrollView
+        contentContainerStyle={{
+          paddingTop: headerHeight,
+          paddingBottom: tabBarHeight,
+        }}
+      >
+        <Albums scrollEnabled={false} />
+      </ScrollView>
+    </>
+  );
+};
+
+const Tab = createBottomTabNavigator<BottomTabParams>();
+
+export function BottomTabsAnimated({
+  navigation,
+  route,
+}: StackScreenProps<ParamListBase, string>) {
+  const routeName = getFocusedRouteNameFromRoute(route) ?? 'Article';
+
+  React.useLayoutEffect(() => {
+    navigation.setOptions({
+      headerShown: false,
+      title: routeName,
+    });
+  }, [navigation, routeName]);
+
+  return (
+    <Tab.Navigator
+      screenOptions={{
+        headerLeft: (props) => (
+          <HeaderBackButton {...props} onPress={navigation.goBack} />
+        ),
+
+        animationEnabled: true,
+        transitionSpec: TransitionSpecs.CrossFadeAnimationSpec,
+        styleInterpolator: ({ current }) =>
+          SceneStyleInterpolator.forCrossFade({ current }),
+      }}
+    >
+      <Tab.Screen
+        name="TabStack"
+        component={SimpleStack}
+        options={{
+          title: 'Article',
+          tabBarIcon: getTabBarIcon('file-document'),
+        }}
+      />
+      <Tab.Screen
+        name="TabChat"
+        component={Chat}
+        options={{
+          tabBarLabel: 'Chat',
+          tabBarIcon: getTabBarIcon('message-reply'),
+          tabBarBadge: 2,
+        }}
+      />
+      <Tab.Screen
+        name="TabContacts"
+        component={Contacts}
+        options={{
+          title: 'Contacts',
+          tabBarIcon: getTabBarIcon('contacts'),
+        }}
+      />
+      <Tab.Screen
+        name="TabAlbums"
+        component={AlbumsScreen}
+        options={{
+          title: 'Albums',
+          headerTintColor: '#fff',
+          headerTransparent: true,
+          headerBackground: () => (
+            <BlurView
+              tint="dark"
+              intensity={100}
+              style={StyleSheet.absoluteFill}
+            />
+          ),
+          tabBarIcon: getTabBarIcon('image-album'),
+          tabBarInactiveTintColor: 'rgba(255, 255, 255, 0.5)',
+          tabBarActiveTintColor: '#fff',
+          tabBarStyle: {
+            position: 'absolute',
+            borderTopColor: 'rgba(0, 0, 0, .2)',
+          },
+          tabBarBackground: () => (
+            <BlurView
+              tint="dark"
+              intensity={100}
+              style={StyleSheet.absoluteFill}
+            />
+          ),
+        }}
+      />
+    </Tab.Navigator>
```

Comment:
This navigator can be simplified by removing any customization unrelated to animations (such as blurred tab bar).